src/App.js

```diff
@@ -1,79 +1,15 @@
 import React from 'react';
 import './App.css';
+import { Welcome } from './components/welcome/welcome';
+import { Article } from './components/article/article';
+import { Header } from './components/header/header';
```

Comment:
компоненты называют с большой буквы
```suggestion
import { Welcome } from './components/Welcome/Welcome';
import { Article } from './components/Article/Article';
import { Header } from './components/Header/Header';
```